ci/pipelines/deploy.yml

```diff
@@ -1167,7 +1167,12 @@ jobs:
           <<: *cf-creds
           CF_SPACE: smoke-tests
           COMMAND: cf ssh smoke-tests-staging -c /app/bin/smoke-cardp1
-
+        on_success:
+          task: notify hg
+          file: omnibus/ci/tasks/send-hg-metric.yml
+          params: 
+            METRIC_NAME: ci.concourse.hosted-graphite.test
```

Comment:
Have we thought what the naming convention will be? I assume this is just a place holder :)